packages/gestalt-design-tokens/tokens/classic/base-color-searchguide-light.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
let's not touch the classic folder. this is component tokens within the base folder. let's move this values to the comp-web-color-light.json

Also, we need a **_comp-web-color-dark.json_** file too.

Reply:
Done